docs/notebooks/ensembles.ipynb

```diff
@@ -17,8 +17,6 @@
     "\n",
     "from __future__ import annotations\n",
     "\n",
-    "from pathlib import Path\n",
-    "\n",
```

Comment:
This was just unused? This is unrelated to the general idea of the PR correct?

Reply:
Correct, haha. Bad habits die hard.

Reply:
Having this small interaction just to make sure is still more efficient than having a whole PR for such a small change IMO. Maybe such changes should be grouped, and maybe not the best practice, but there are certainly pros and cons

Reply:
As the code base is getting more and more cleaned up, I feel that this is becoming less and less of a concern, thankfully. But you're right that it would be good to have a comprehensive/in-depth plunge into the code to see if there are any more tiny bugs/issue worth cleaning up.